deps/v8/gypfiles/v8.gyp

```diff
@@ -2023,20 +2021,28 @@
             'sources': [
               '../src/base/debug/stack_trace_posix.cc',
               '../src/base/platform/platform-linux.cc',
-              '../src/base/platform/platform-posix.h',
               '../src/base/platform/platform-posix.cc',
-              '../src/base/platform/platform-posix-time.h',
+              '../src/base/platform/platform-posix.h',
               '../src/base/platform/platform-posix-time.cc',
+              '../src/base/platform/platform-posix-time.h',
             ],
           }
         ],
+        ['OS=="aix"', {
+          'sources': [
+            '../src/base/debug/stack_trace_posix.cc',
+            '../src/base/platform/platform-aix.cc',
+            '../src/base/platform/platform-posix.cc'
```

Comment:
Missing trailing comma?

Reply:
Thanks!

Reply:
Nice :). I wonder how gyp interpreted this. Was it even valid syntax?

Reply:
Yeah, one of the problems of `.gyp` files is that this is valid python syntax (concat the two strings).
You can run the following monstrosity to get GYP to check the syntex:
```
python tools\gyp\gyp_main.py --check deps\v8\gypfiles\v8.gyp -I common.gypi -I config.gypi --depth=.
```
or for the whole project
```
python tools\gyp\gyp_main.py --check node.gyp -I common.gypi -I config.gypi --depth=.
```